GIT-VERSION-GEN

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 GVF=GIT-VERSION-FILE
-DEF_VER=v2.20.0
+DEF_VER=v2.20.0.gvfs.1.1
```

Comment:
@jrbriggs @kewillford @dscho Did we want this to be `.vfs.1.1`? In that case, I can replay the rebase to do it right.

Reply:
I did it that way (using `vfs` instead of `gvfs` in https://github.com/Microsoft/git/pull/81... and @kewillford said that he wanted to have that change, and tested it AFAICT.

Reply:
Yes the other change I had to make was on line 15 see https://github.com/Microsoft/git/commit/b49bac0e4c8b6623bdcd19ded1555443c2e7c09b